activerecord/lib/active_record/relation/finder_methods.rb

```diff
@@ -107,7 +107,8 @@ def take!
     end
 
     # Find the first record (or first N records if a parameter is supplied).
-    # If no order is defined it will order by primary key.
+    # If no order is defined it will order by primary key if the collection is not loaded.
+    # If the collection is loaded it will get the first record from that collection.
```

Comment:
What does "that collection" refer to here? Why would I care whether it's loaded or not? The wording here seems to imply very intimate knowledge of Active Record, maybe we should be more explicit for people who are less familiar?